test/experiments/parse_commandline.jl

```diff
@@ -18,6 +18,14 @@ function parse_commandline()
         help = "Precipitation model choice: NoPrecipitation, Precipitation0M, Precipitation1M"
         arg_type = String
         default = "Precipitation1M"
+        "--rain_formation_scheme_choice"
+        help = "Rain formation scheme choice: CliMA_1M, KK2000, B1994, TC1980, LD2004"
+        arg_type = String
+        default = "CliMA_1M"
+        "--prescribed_Nd"
+        help = "Prescribed number of cloud droplets (used in KK2000, B1994, TC1980 and LD2004 rain formation schemes"
+        arg_type = Real
+        default = 1e8
```

Comment:
I would prefer to keep everything in the code in SI units and do all the conversions in post processing. Also because all our parameters are defined in SI